codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -959,4 +940,139 @@ private boolean isLocal() {
             return this == localEndpoint;
         }
     }
+
+    /**
+     * Provides an interface to manage the active streams for this {@link Http2Connection}.
+     */
+    private interface ActiveStreams {
```

Comment:
@Scottmitch why is that? Should be able to put the `Event` interface into `DefaultActiveStreams`?